README.md

```diff
@@ -21,13 +21,25 @@ list in the terminal. See it in action:
     >>> options = ['Java', 'JavaScript', 'Python', 'PHP', 'C++', 'Erlang', 'Haskell']
     >>> option, index = pick(options, title)
 
+
+**pick** multiselect example:
+
+    >>> from pick import pick
+
+    >>> title = 'Please choose your favorite programming language (press SPACE to mark, ENTER to continue): '
+    >>> options = ['Java', 'JavaScript', 'Python', 'PHP', 'C++', 'Erlang', 'Haskell']
+    >>> selected = pick(options, title, multi_select=True)
+    >>> print selected
```

Comment:
we can add a comment of the sample output of `selected`, so the user can know the structure of it